book/src/internals/build-system.md

```diff
@@ -0,0 +1,100 @@
+<!--
+SPDX-FileCopyrightText: 2024 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+SPDX-FileContributor: Leon Matthes <leon.matthes@kdab.com>
+
+SPDX-License-Identifier: MIT OR Apache-2.0
+-->
+
+# The CXX-Qt Build System
+
+Building with CXX-Qt is somewhat more complicated than it may sound at first.
+
+**STATUS:** In-progress - this is a draft, not entirely implemented yet!
```

Comment:
is it still draft? most of it has landed now?

Reply:
Good point, I removed the note.